packages/manager/apps/catalog/mocks/catalog.ts

```diff
@@ -0,0 +1,17 @@
+import { Handler } from '../../../../../playwright-helpers';
```

Comment:
Same as previous

Reply:
No it doesn't work
The alias in tsconfig.json is not taken into account